pkgs/tools/networking/openconnect/common.nix

```diff
@@ -32,7 +34,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gmp libxml2 stoken zlib (if useOpenSSL then openssl else gnutls) ]
     ++ lib.optional stdenv.isDarwin PCSC
-    ++ lib.optional stdenv.isLinux p11-kit;
+    ++ lib.optional stdenv.isLinux p11-kit
+    ++ lib.optional (stdenv.isLinux && useDefaultExternalBrowser) xdg-utils;
```

Comment:
Why only on linux?

This PR rebuilds a lot of packages which means we must target staging. Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging) to not potentially ping a lot of people.

Reply:
`xdg-utils` is not normally something you would want on macOS since the macOS ecosystem does not follow FreeDesktopOrg practices. Also, I don't know of any utility on macOS that consumes `libopenconnect`. On Linux, both `NetworkManager-openconnect` and `plasma-nm` do. That said, it is possible to enable `DEFAULT_EXTERNAL_BROWSER` support using macOS utility `open` but there would be no way to test since there would be no application available that consumes this feature. The `openconnect` CLI does not consume this feature.